### ISSUE TITLE
Aurora Windows wallet is in endless loop trying to unlock wallet

### ISSUE THREAD

**Issue by finddabug** [pura-dev/pura-release#2]
When starting the Pura Aurora 1.1.0 Beta Windows wallet the window displays "Wallet is encrypted - Loading please wait" then the app goes into a white screen and the same message is displayed again after a few seconds. The password is 100% correct since Pura Core works just fine with the HD encrypted wallet.

**Comment by pura-admin** [comment]
This issue has been corrected. Please see v1.1.1

**Comment by finddabug** [comment]
Unfortunately this issue still exists on my Windows 10 X64 Pura wallet with HD encryption. I can put in the password click the arrow and Aurora displays: Unlocking wallet but nothing happens :-(
As an additional problem I noticed now that my wallet database gets corrupted in the process. When going back to Pura Core wallet 1.0.0 wallet starts up with the error message: ` Wallet corrupted - rebuilding database..` Had to move my Puras to other wallet; delete everything and reinstall Pura Core from scratch..than move the Puras back into the new wallet :-(

**Comment by pura-admin** [comment]
Unfortunately if the wallet is corrupted in the old wallet, it will continue to be corrupt in the new wallet. The new wallet relies on the old RPC to handle things like this, so the behavior should be identical. We are absorbing RPC functionality into the Aurora build between now and Feb 2018. I'm curious though, you had wallet corruption issues after reinstalling and using the new wallet? If so, I can give you some directories and registry entries to look out for when rebuilding.

**Comment by finddabug** [comment]
To be clear my Pura Core wallet was A-OK before I installed Aurora 1.1.1. Than I installed Aurora and tried to unlock the wallet. Unsuccessful. Than I went back to Pura Core and now my wallet was corrupted. But Pura Core was able to restore it again. TG because my backup wallet was a bit older. Anyways I uninstalled Aurora since Windows is not really my operation system of choice when it comes to my crypto holdings.
